ice-rest-catalog/src/main/java/com/altinity/ice/rest/catalog/internal/maintenance/OrphanFileScanner.java

```diff
@@ -50,6 +54,14 @@ private Set<String> getAllKnownFiles() {
       FileIO io = table.io();
       for (ManifestFile manifest : snapshot.dataManifests(io)) {
         knownFiles.add(manifest.path());
+        // Add data files inside each manifest
+        try (CloseableIterable<DataFile> files = ManifestFiles.read(manifest, table.io())) {
+          for (DataFile dataFile : files) {
+            knownFiles.add(dataFile.path().toString());
+          }
+        } catch (Exception e) {
+          logger.error("Error getting list of data files", e);
```

Comment:
network error here leads to us deleting files from catalog that are not supposed to be deleted